allure-report-face/src/main/webapp/js/controllers.js

```diff
@@ -51,6 +51,9 @@ angular.module('allure.core.controllers', [])
         }, {
             name: "РУС",
             locale: "ru"
+        },{
+           name: "PT-BR",
```

Comment:
Use full name here.
I've changed this field for other languages in `master` branch